build.gradle

```
@@ -4,33 +4,50 @@ plugins {
 	id 'io.spring.dependency-management' version '1.0.15.RELEASE'
 }
 
-group = 'com'
-version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
 
+allprojects {
+	group = 'com'
```

group name 의미없는 것보다는 의미있게  변경해도 좋을 것 같습니다!
```suggestion
	group = 'hobbyloop'
```